common/src/primitives/consensus_data.rs

```diff
@@ -8,7 +8,6 @@ pub enum ConsensusData {
     #[codec(index = 1)]
     PoW(PoWData),
 }
-
```

Comment:
Is there any reason this blank line has been removed? It looked neater before IMO

Reply:
You're right. There was a lot of code there which I had to remove, overlooked that line!